src/meta/ClusterInfo.cpp

```diff
@@ -409,7 +431,8 @@ std::shared_ptr<TableSpec> loadTable(std::string name, const YAML::Node& td) {
       asTimeSpec(td["time"]),
       asAccessRules(td["access"]),
       asBucketInfo(td["bucket"]),
-      asSettings(td["settings"]));
+      asSettings(td["settings"]),
+      asMacroValues(td["path_macros"]));
```

Comment:
we have an undocumented convention of trying to use a single word at the top config as much as we could. instead of `path_macros`, can we just use `macros` if there are no better alternatives?